### PR TITLE
after modify

### PR DESCRIPTION
**Pull request by mazooo29**
the modify of yesterday work

### CODE REVIEW

File: CSS/tabletSize.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@media screen and (min-width:426px) and (max-width:768px){
```
Comment (IbrahimLama) [inline CSS/tabletSize.css]: no need for this media query

**Comment by mazooo29** [comment]
But there are the other size of the laptop how i do it

File: CSS/tabletSize.css (hunk 1 above)
Comment (IbrahimLama) [reply CSS/tabletSize.css]: the desktop will be handle when you add a min-width media query for it, and the code outside the media queries will be applied to the mobile

**Comment by mazooo29** [comment]
the conflict problem

**Pull request closed without merging**